testutil/src/main/java/org/spine3/test/NullToleranceTest.java

```diff
@@ -271,23 +273,16 @@ private static void checkException(Throwable cause) {
         /**
          * Checks the stack trace elements.
          *
-         * <p> According to the business rules the non-null check should be in the each utility method.
-         * So the usage of the {@link Preconditions}#checkNotNull method should allocated there.
+         * <p> It is expected that each of the tested utility methods invokes
+         * {@link Preconditions}#checkNotNull(<arg types here>) as a first step of the execution.
```

Comment:
`<arg types here>` is a placeholder. You should have replaced it with the method you actually refer to.
Please check the syntax of `{@link}` when referring a static method of the class.